Makefile

```diff
@@ -10,7 +10,7 @@ ibctest: gen ## Build ibctest binary into ./bin
 
 .PHONY: test
 test: ## Run unit tests
-	@go test -cover -short -race -timeout=60s $(shell go list ./... | grep -v /cmd/)
+	@go test -cover -short -race -timeout=60s ./...
```

Comment:
`testing.Short()` and `-short` flag means we can run unit tests throughout the whole project. No need to filter out packages.